bindings/ros/aditof_roscpp/include/aditof_roscpp/aditof_utils.h

```diff
@@ -0,0 +1,22 @@
+#ifndef ADITOF_UTILS_H
+#define ADITOF_UTILS_H
+
+#include <aditof/camera.h>
+#include <aditof/camera_96tof1_specifics.h>
```

Comment:
This is no required here, in this header. Also not system, glog and probably not ros.